packages/block-library/src/cover/edit.js

```diff
@@ -1,6 +1,7 @@
 /**
  * External dependencies
  */
+import { noop } from 'lodash';
 import classnames from 'classnames';
 import FastAverageColor from 'fast-average-color';
```

Comment:
Hi @ItsJonQ, the cover block was already computing the color of an image using the FastAverageColor module. I think we should avoid using two different modules for the same task so I guess we should opt for one of the modules and update the code to use the chosen module.

Reply:
@jorgefilipecosta Oh! I wasn't aware of that. Thank you for pointing that out. I'll make an update